### ISSUE TITLE
fix xtend android maven archetype

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#18]

**Comment by cdietrich** [comment]
the error occurs when building the maven project generated by the archetype and looks like http://stackoverflow.com/questions/27700099/error-injecting-com-jayway-maven-plugins-android-phase01generatesources-generat

**Comment by cdietrich** [comment]
maybe we should use 

```
					<groupId>com.simpligility.maven.plugins</groupId>
					<artifactId>android-maven-plugin</artifactId>
```
+ a std maven layout

**Comment by spoenemann** [comment]
Seems to be fixed with #93.
